demo/minimal-api.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <button id="btn" type="button">Copy</button>
+    <script src="../dist-minimal/clipboard.min.js"></script>
+    <script>
+      let btn = document.getElementById('btn');
```

Comment:
It doesn't will cause a great impact, but you can use a `const` to create the BTN instance.

Reply:
Deal, the change can be made.